packages/lib/server/repository/RoutingFormRepository.ts

```diff
@@ -1,6 +1,36 @@
 import prisma from "@calcom/prisma";
+import type { Prisma } from "@calcom/prisma/client";
+
+const defaultSelect = {
+  id: true,
+  description: true,
+  position: true,
+  routes: true,
+  createdAt: true,
+  updatedAt: true,
+  name: true,
+  fields: true,
+  updatedById: true,
+  userId: true,
+  teamId: true,
+  disabled: true,
+  settings: true,
+} as const;
 
 export class RoutingFormRepository {
+  static async findById<T extends Partial<typeof defaultSelect> = typeof defaultSelect>(
+    id: string,
+    options?: {
+      select?: T;
+    }
+  ) {
+    const select = options?.select ?? (defaultSelect as T);
+    return (await prisma.app_RoutingForms_Form.findUnique({
+      where: { id },
+      select,
+    })) as Promise<Prisma.App_RoutingForms_FormGetPayload<{ select: T }> | null>;
```

Comment:
Probably a follow-up but we should return a non-prisma type and also extend this repository from RoutinFormRespository.interface